app/controllers/request-to-go-live/index/post.controller.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+// Local dependencies
+const paths = require('../../../paths')
+
+module.exports = (req, res) => {
+  const externalServiceId = req.service.externalId
+  res.redirect(303, paths.requestToGoLive.index.replace(':externalServiceId', externalServiceId))
```

Comment:
Is this route/ redirect just a placeholder for upcoming work? Not 100% on repo standards but a quick `todo` comment could make this clear and act as something to remove when the work is done.

Reply:
Yes, good idea, I have added `TODO` comments to mark what is next, thank you